tools/ci-ts/test-helpers/chainlinkClient.ts

```diff
@@ -0,0 +1,103 @@
+import { ethers } from 'ethers'
+import execa from 'execa'
+// TODO replace import with CL @types package
+// https://www.pivotaltracker.com/story/show/171715396
+import { JobSpec, JobRun } from '../../../operator_ui/@types/operator_ui'
+import path from 'path'
+import Dockerode from 'dockerode'
+
+const API_CREDENTIALS_PATH = '/run/secrets/apicredentials'
+const docker = new Dockerode()
```

Comment:
This can be either a static property or a private property of the class

Reply:
I would probably prefer this to be passed in as a constructor arg, then you can unit test this class with a mock `Dockerode` instance to check that the right `exec` commands are being formed.

Reply:
It feels a little strange to me to test something that only exists for testing... if this thing breaks we should see our tests fail anyway.

Reply:
Addressed this in: https://github.com/smartcontractkit/chainlink/pull/2506#discussion_r398889452